stores/query_models.go

```diff
@@ -1,8 +1,10 @@
 package stores
 
+import "time"
+
 // QSub are the results of the Qsub query
 type QSub struct {
-	Project      string `bson:"project"`
+	ProjectUUID  string `bson:"project_uuid"`
```

Comment:
For the clients it's not a breaking change. For existing data: migration to a slightly updated schema is necessary